.gitignore

```diff
@@ -39,3 +39,4 @@ uninstaller/UAC.nsh
 *.pyc
 *.pyo
 *.dmp
+*.swp

```

Comment:
nit: Happy to include this, but preferably as a separate commit. We'll just handle this at merge time.